build/root/Makefile

```diff
@@ -123,9 +123,12 @@ endef
 ifeq ($(PRINT_HELP),y)
 verify:
 	echo "$$VERIFY_HELP_INFO"
-else
+else ifeq ($(origin KUBE_VERIFY_GIT_BRANCH), undefined)
```

Comment:
https://github.com/kubernetes/kubernetes/pull/120562#discussion_r1403017393

Used in `hack/verify-govulncheck.sh`